source/dotnet/Library/AdaptiveCards/AdaptiveTypedElement.cs

```diff
@@ -26,10 +28,14 @@ public abstract class AdaptiveTypedElement
         /// Additional properties not found on the default schema
         /// </summary>
         [JsonExtensionData]
-#if !NETSTANDARD1_3
-        [XmlIgnore]
-#endif
+#if NETSTANDARD1_3
         public IDictionary<string, object> AdditionalProperties { get; set; } = new Dictionary<string, object>(StringComparer.OrdinalIgnoreCase);
+#else
+        // Dictionary<> is not supported with XmlSerialization because Dictionary is not serializable, SerializableDictionary<> is
+        [XmlElement]
+        public SerializableDictionary<string, object> AdditionalProperties { get; set; } = new SerializableDictionary<string, object>(StringComparer.OrdinalIgnoreCase);
```

Comment:
I'm just guessing here but wanted to throw this out before I closed the tab: Would changing this property type to `IDictionary<string, object>` fix the breaking change report? The fact that it happens to be implemented by a `SerializableDictionary` doesn't seem super relevant. Presumably this should have had a private setter anyway (typically people would just add/remove/clear the dictionary, not assign one wholesale)

Reply:
So, no because we end up in the same boat -- The type info is significant because that's what ends up getting reflected on in the call to XmlSerializer() .. @tomlm sent you a mail offline around removing this change from the release/1.2 branch since the change in and of itself blocks upgrade to 1.2.3 by Cortana skills ..

Reply:
I am evaluating using the DataContractSerializer approach here as well since that would mean not needing the custom implmentation of the SerializableDictionary but for now am just going to roll this change back from release/1.2